### DISCUSSION TITLE
Question: Stretching Flyout Content without Scrollviewer to Max Width

### DISCUSSION THREAD

**Discussion by AtosNicoS** [microsoft/microsoft-ui-xaml#7474] [Q&A]
I basically ran into this issue:
https://social.msdn.microsoft.com/Forums/en-US/ef4b2b75-6356-42d1-8a6d-434ee56c8c0d/why-does-the-flyout-not-resize-sensibly?forum=w81prevwCsharp

I found out, that this is a possible solution to fix it:
```xaml
<Flyout.FlyoutPresenterStyle>
    <!--Disable infinite flyout width-->
    <Style TargetType="FlyoutPresenter">
        <Setter Property="ScrollViewer.HorizontalScrollMode" Value="Disabled" />
        <Setter Property="ScrollViewer.HorizontalScrollBarVisibility" Value="Disabled" />
    </Style>
</Flyout.FlyoutPresenterStyle>
```

But why does the following not work? Is this a bug?
```xaml
<Flyout ScrollViewer.HorizontalScrollBarVisibility="Disabled" ScrollViewer.HorizontalScrollMode="Disabled">
```

The related theme file is:
https://github.com/microsoft/microsoft-ui-xaml/blob/release/2.5/dev/CommonStyles/FlyoutPresenter_themeresources.xaml

**Comment by StephenLPeters** [comment]
Flyout is not a control, it is just a dependency object that creates a popup that is inserted into the tree. The root of the content is a FlyoutPresenter with the style FlyouPresenterStyle  The default FlyoutPresenterStyle template binds the ScrollViewer attached properties to the scroll viewer in the template, however there isn't any code that forwords these attach property from the flyout to the flyout presenter.  We could add that code, and probably should, we forward some other poperties.
